html/captive-portal/templates/lost_stolen.html

```diff
@@ -0,0 +1,40 @@
+[% IF status == "success" %]
+<div class="media media--notice u-p u-mb">
+  <div class="media__img">[% flashIcon(level='notice') %]</div>
+  <p class="media__body">[% i18n_format("Your device %s has been declared as lost or stolen.", mac) | none %]</p>
+</div>
+[% ELSIF status == "error" %]
+<div class="media media--error u-p u-mb">
+  <div class="media__img">[% flashIcon(level='error') %]</div>
+  <p class="media__body">[% i18n_format("An error occured while trying to delcare the device %s as lost or stolen.", mac) | none %]</p>
+</div>
+[% ELSIF status == "notowned" %]
+<div class="media media--error u-p u-mb">
+  <div class="media__img">[% flashIcon(level='error') %]</div>
+  <p class="media__body">[% i18n_format("You do not own this device %s, therefor you can not decalre it as lost or stolen.", mac) | none %]</p>
+</div>
+[% END %]
+<div class="layout layout--center">
```

Comment:
Please put the menu in a partial that you reuse across pages so that there is no duplication